modules/subgraphs/gauge-subgraph/gauge-subgraph-queries.graphql

```diff
@@ -155,3 +155,47 @@ query VotingEscrowLocks(
         }
     }
 }
+
+query GaugesInfo($poolIds: [Bytes!]) {
+    liquidityGauges(
+        where: { poolId_in: $poolIds, gauge_not: null }
+        # TODO: add pagination
```

Comment:
We currently have less than 200 results in these queries. 

Will we ever reach 1000 results? We have 3 options here: 

1) Postpone the pagination decision (YAGNI?) by adding an exception when we detect for instance 900 results.  If that exception is thrown in the future it means that it is time to implement pagination
2) Implement pagination in GQL queries
3) Implement pagination in TS services

Reply:
We implement pagingation in the TS service, see `getAllGaugeShares` for reference